yt/visualization/plot_container.py

```diff
@@ -208,6 +215,11 @@ class PlotContainer:
     _plot_type = None
     _plot_valid = False
 
+    # Plot defaults
+    _colormap_config: DictWithFactory = None
+    _log_config: DictWithFactory = None
+    _units_config: DictWithFactory = None
```

Comment:
you'll need to add `from typing import Optional`, this is needed to satisfy type checkers when a type is declared but the default value is `None`
```suggestion
    _colormap_config: Optional[DictWithFactory] = None
    _log_config: Optional[DictWithFactory] = None
    _units_config: Optional[DictWithFactory] = None
```